src/atoms/Text/index.js

```diff
@@ -5,19 +5,77 @@ import classnames from 'classnames';
 import colors from 'atoms/Color/colors.module.scss';
 import styles from './text.module.scss';
 
+const convertChild = (child) => {
+  // Check children for other TextComponents
+  // and automatically convert them into <spans> if they are <p>.
+  if (child.type === Text && child.props.tag === 'p') {
+    return React.cloneElement(child, { tag: 'span' });
+  }
+
+  return child;
+};
+
+function Em(props) {
+  const newProps = Object.assign({}, props, {
+    tag: 'em',
+  });
+
+  return Text(newProps);
+}
+
+function Strong(props) {
+  const newProps = Object.assign({}, props, {
+    tag: 'strong',
+  });
+
+  return Text(newProps);
+}
+
+function Strike(props) {
+  const newProps = Object.assign({}, props, {
+    tag: 'span',
+    variant: 'strikethrough'
+  });
+
+  return Text(newProps);
+}
+
+function Sup(props) {
+  const newProps = Object.assign({}, props, {
+    tag: 'sup',
+    variant: 'sup'
```

Comment:
@patrickkim This will create duplicate class, I believe, since each Text component gets a styles[tag] and a styles[variant]